components/Avatar/avatar.css

```diff
@@ -0,0 +1,62 @@
+@import '/css/main.css';
+
+.avatar{
+    border-radius: 50%;
+    display: flex;
+    justify-content: space-around;
```

Comment:
Use justify-content: center as you want to center the text. This also works correctly but center would be more sensible to use here

Reply:
yes It would be updated, right now I am just comparing this with my another components bur before final push to the I will update it to the center